spec/features/viewing_chitter_spec.rb

```diff
@@ -0,0 +1,17 @@
+feature 'Viewing Chitter' do
+  scenario 'visiting the index page' do
+    visit('/')
+    expect(page).to have_content "Chitter Away!"
+  end
+end
+
+feature 'Viewing Peeps on Chitter' do
+  scenario 'A user can see a list of peeps' do
+    visit('/peeps')
+    
+    expect(page).to have_content "hello everyone"
+    expect(page).to have_content "Still got this bloomin lurgy and high temperature."
+    expect(page).to have_content "Why is he sniffing so much?"
+    expect(page).to have_content "I am in the curry house"
```

Comment:
I think as you have shown time in the views, you can also expect(page) to have the date as content. That would be a proof that every variable you decide to show is actually showing.